src/Classes/ServiceAPI/MyRadio_Show.php

```diff
@@ -586,20 +586,6 @@ public function getGenre()
         return isset($this->genres[0]) ? $this->genres[0] : null;
     }
 
-    public function isCurrentUserAnOwner()
```

Comment:
MyRadio_{Show,Season,Timeslot} are instances of MyRadio_Metadata_Common which use MyRadio_Creditable